Add visibility switch to `Settings > Writing` password field.

costdev opened the pull request:
Trac ticket: https://core.trac.wordpress.org/ticket/9883

Commit:
Add visibility switch to password field.

Commit:
Add selectors to existing variables.

Commit:
Allow a visibility switch with no strength meter.

Commit:
Make the reveal-toggle consistent with login form.

peterwilsoncc commented on src/wp-admin/options-writing.php:
```diff
@@ -163,11 +165,21 @@
+	<td>
+		<input class="hidden" value=" " /><!-- #24364 workaround -->
+		<span class="wp-pwd">
+			<input type="text" name="mailserver_pass" id="mailserver_pass" class="regular-text" autocomplete="off" data-reveal="1" data-pw="<?php echo esc_attr( get_option( 'mailserver_pass' ) ); ?>" />
```
```suggestion
			<input type="text" name="mailserver_pass" id="mailserver_pass" class="regular-text ltr" autocomplete="off" data-reveal="1" data-pw="<?php echo esc_attr( get_option( 'mailserver_pass' ) ); ?>" />
```

Per @sabernhardt's comment on the ticket.

peterwilsoncc reviewed (commented):
On the options page, I am seeing a JavaScript error in the console `Uncaught TypeError: can't access property "data", $toggleButton is undefined` and the toggle button is not working. 

The user-profile javascript also includes code for setting the colour scheme on a users profile. These settings aren't available on the settings page so it seems a little hacking to include the JavaScript. 

It's yet another JavaScript file but given how much the password toggle is used in core, it may be worth moving it in to a separate file.

Commit:
Force LTR on the password input field and button.

Commit:
Add `ltr` class to password input field.

Commit:
Disable `#mailserver_pass` focus during binding.

Previously, the password input on Options > Writing was focused on load.
The `#mailserver_pass` input field is now excluded from this behaviour.

Commit:
Fix alignment for password manager icons.

Commit:
Fix alignment for password manager icons in both states.
